### PR TITLE
fix problem with spaces in workdir path

### PR DESCRIPTION
**Pull request by cvgs**
If workdir contains spaces, the four "find" commands below would fail.

### CODE REVIEW

**Comment by cvgs** [comment]
Hi Graham,
yeah, sorry for „spamming“ you with all my commits; apparently i am too optimistic and then always find the odd thing to re-fix again. So if you should decide to accept those Pull requests i guess squashing the individual commits would make the final commit history a bit more readable. :-)

I’ll be happy to adapt the CHANGELOG.md for each individual Pull request.

Otherwise i guess i’m done with changes for the time being; the script with all changes merged into it works for me. i wanted to add the removal of the package receipt in the cleanup stage, but that needs to wait for a bit, and it seems to be more than a one-line change.

Thanks for all your efforts!

**Comment by cvgs** [comment]
I added all the changes to this CHANGELOG.md, because having them listed in each individual Pull request would cause merge failures.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ No date
 
 - Universal python build packages.
 - Use `pkg_resources` instead of `distutils` where available (allows for removal if `distutils` in python 3.12 - addresses [grahampugh/macadmin-scripts/issues/47](https://github.com/grahampugh/macadmin-scripts/issues/47)).
+- Allows the usage of spaces in `--workdir` and `--path`.
+- Fixed an issue with `--max-password-attempts=NN`.
```
Comment (grahampugh) [inline CHANGELOG.md:14]: `--max-password-attempts` is new in 26.1, and have not yet released a full version, so this change note is not required.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ No date
...
+- Fixed an issue with the first password check always failing.
```
Comment (grahampugh) [inline CHANGELOG.md:15]: Similarly, not something that existed or failed prior to 26.1

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,14 @@ No date
...
+- Changes dialogs so that the cancel button is on the left, and default button is on the right.
```
Comment (grahampugh) [inline CHANGELOG.md:16]: My comment about the apparent changed default needs to be resolved.

Comment (grahampugh) [reply CHANGELOG.md:15]: Clarified.

**Pull request merged**